src/wp-admin/options-general.php

```diff
@@ -160,13 +204,10 @@ class="<?php echo esc_attr( $classes_for_button ); ?>"
 		</button>
 	</div>
 
-	<p class="description">
-		<?php _e( 'Site Icons are what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. Upload one here!' ); ?>
-	</p>
 	<p class="description">
 		<?php
 			/* translators: %s: Site Icon size in pixels. */
-			printf( __( 'Site Icons should be square and at least %s pixels.' ), '<strong>512 &times; 512</strong>' );
+			printf( __( 'Site Icons are what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. They should be square and at least %s pixels.' ), '<strong>512 &times; 512</strong>' );
```

Comment:
Maybe wrap size string in a `<code>`-tag instead to align with other descriptive texts on the page?

![bild](https://github.com/WordPress/wordpress-develop/assets/11491369/4c6b564d-8396-46d9-b180-56468fa7ef83)